### PR TITLE
Add junit test and CI

### PR DESCRIPTION
**Pull request by Placu95**
The reason of CI failure is the protelis program `distanceTo`, that is incompatible with protelis 13.
Is there a change log to understand how to modify the program?

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,11 +1,23 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.3.41"
```
Comment (DanySK) [inline build.gradle]: There should be a built-in shortcut for that, maybe `kotlin("jvm")`?

File: build.gradle [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
+            srcDir "src/main/protelis"
+        }
+    }
+}
+
 dependencies {
-	compile("it.unibo.alchemist:alchemist:$alchemistVersion") {
-		exclude module: 'org.eclipse.xtext.dependencies'
-	}
+   compile("it.unibo.alchemist:alchemist:$alchemistVersion") {
```
Comment (DanySK) [inline build.gradle]: This is deprecated, use `implementation`

File: build.gradle [hunk 3]
```diff
@@ -1,11 +1,23 @@
...
+       exclude module: 'org.eclipse.xtext.dependencies'
```
Comment (DanySK) [inline build.gradle]: There should be no need to exclude this one, can you test without the exclusion?

File: build.gradle (hunk 1 above)
Comment (Placu95) [reply build.gradle]: No, this shortcut is only for gradle kts

File: build.gradle (hunk 2 above)
Comment (Placu95) [reply build.gradle]: done

File: build.gradle (hunk 3 above)
Comment (Placu95) [reply build.gradle]: done

File: build.gradle (hunk 1 above)
Comment (DanySK) [reply build.gradle]: OK. I'm merging now, but next step: convert to kts.

**Comment by DanySK** [comment]
Looks good now, merging

**Comment by DanySK** [comment]
Next step @Placu95: switch to Gradle kts

**Pull request merged**